src/mod_openssl.c

```diff
@@ -546,6 +551,163 @@ network_openssl_ssl_conf_cmd (server *srv, plugin_config *s)
   #endif
 }
 
+#if defined(HAVE_WOLFSSL_SSL_H) && defined(HAVE_PK_CALLBACKS)
```

Comment:
Please name this specific to wolfSSL, e.g. HAVE_WOLFSSL_PKCALLBACKS
(How does --enable-pkcallbacks turn into HAVE_WOLFSSL_PK_CALLBACKS?)